rllib/agents/trainer_template.py

```diff
@@ -186,6 +187,20 @@ def step(self):
                     assert isinstance(evaluation_metrics, dict), \
                         "_evaluate() needs to return a dict."
                     res.update(evaluation_metrics)
+
+            # Check `env_task_fn` for possible update of the env's task.
+            if self.config["env_task_fn"] is not None:
+                assert callable(self.config["env_task_fn"])
```

Comment:
assertions in code can be a bit unwieldy, especially when the users see it. 

I would consider writing a special check function that also prints a user friendly message.

Reply:
done